src/program.js

```diff
@@ -103,7 +103,9 @@ export class Program {
       await runCommand(argv);
     } catch (error) {
       const prefix = cmd ? `${cmd}: ` : '';
-      log.error(`\n${prefix}${error.stack}\n`);
+      if (!(error instanceof UsageError) || argv.verbose) {
+        log.error(`\n${prefix}${error.stack}\n`);
+      }
```

Comment:
this needs an `else` block that prints the error without any stack information. For example:

    log.error(`\n${prefix}${error}\n`);